src/model/PersonalAccessToken.ts

```diff
@@ -1,6 +1,6 @@
 export class PersonalAccessToken {
     public token: string = ""
     public description: string = ""
-    public createdAt: number = Date.now()
-    public expiresAt: number = Date.now()
+    public created: number = Date.now()
+    public expiry: number = Date.now()
```

Comment:
These would need to be integers to support UNIX epoch timestamps, unless we plan to manually convert the values to/from Date objects each time!

Reply:
I'm not sure what you mean here.
For [reference](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/now) Date.now() returns an integer, the number of ms from 1970 til now.

Reply:
Oh, you're right. I mistook this for `new Date()` which returns a `Date` object instead. My bad, please ignore!